src/ViewInstance.js

```diff
@@ -434,7 +440,7 @@ class ViewInstance extends React.Component {
           </IfInterface>
 
           {
-            canEditDeriveMARCRecord && (
+            (canCreateMARCHoldingsForInstanceWithSourceMARC || canEditDeriveMARCRecord) && (
```

Comment:
We don't need to change anything with `View source` option because it's not related to Holdings. I think we should render it when `canEditDeriveMARCRecord` is `true` and that's it

Reply:
Yeah, you are right. Will fix it, thanks.